packages/icons/src/svg/sharing-user.svg

```diff
@@ -1,8 +1,4 @@
-  </g>
-  <g class="tsharing-user-chip" fill="#66BDFF">
-    <path d="M6.8607,7.92070026 C6.8607,9.60470026 5.4957,10.9697003 3.8117,10.9697003 C2.1277,10.9697003 0.7627,9.60470026 0.7627,7.92070026 C0.7627,6.23670026 2.1277,4.87170026 3.8117,4.87170026 C5.4957,4.87170026 6.8607,6.23670026 6.8607,7.92070026"/>
-  </g>
+    <path d="M14.356504,10.7050861 C13.257504,9.49608606 11.387504,9.40608606 10.177504,10.5040861 L6.07350397,8.38208606 C6.07950397,8.30008606 6.09750397,8.22408606 6.09750397,8.14008606 C6.09750397,8.05708606 6.07950397,7.98008606 6.07350397,7.89908606 L9.18950397,6.28708606 C9.88950397,7.17508606 10.961504,7.75608606 12.180504,7.75608606 C14.290504,7.75608606 16.000504,6.04608606 16.000504,3.93608606 C16.000504,1.82708606 14.290504,0.11608606 12.180504,0.11608606 C10.070504,0.11608606 8.36050397,1.82708606 8.36050397,3.93608606 C8.36050397,3.98308606 8.37250397,4.02708606 8.37450397,4.07308606 L5.05650397,5.78808606 C3.91950397,4.73208606 2.15950397,4.73208606 1.02250397,5.78808606 C-0.234496035,6.96208606 -0.347496035,8.91608606 0.766503965,10.2260861 C0.796503965,10.2630861 0.828503965,10.2990861 0.860503965,10.3340861 C1.97650397,11.5360861 3.85450397,11.6070861 5.05650397,10.4920861 L9.16150397,12.6160861 C9.15450397,12.6960861 9.13650397,12.7730861 9.13650397,12.8560861 C9.13250397,13.6110861 9.39550397,14.3440861 9.87950397,14.9230861 C9.91050397,14.9600861 9.94050397,14.9960861 9.97250397,15.0310861 C11.071504,16.2400861 12.941504,16.3300861 14.151504,15.2310861 C15.414504,14.0740861 15.545504,12.1290861 14.449504,10.8120861 C14.419504,10.7760861 14.388504,10.7400861 14.356504,10.7050861"/>
+    <path fill="#66BDFF" d="M15.235904,3.93608606 C15.235904,5.62408606 13.867904,6.99208606 12.179904,6.99208606 C10.491904,6.99208606 9.12390397,5.62408606 9.12390397,3.93608606 C9.12390397,2.24908606 10.491904,0.88008606 12.179904,0.88008606 C13.867904,0.88008606 15.235904,2.24908606 15.235904,3.93608606"/>
```

Comment:
It's normal to have this color on the svg, it's a colored svg. This color cannot change on the application.